File: support_tools/swigwin-3.0.12/Examples/test-suite/python/overload_extend_runme.py
import overload_extend

f = overload_extend.Foo()
if f.test() != 0:
    raise RuntimeError
if f.test(3) != 1:
    raise RuntimeError
if f.test("hello") != 2:
    raise RuntimeError
if f.test(3, 2) != 5:
    raise RuntimeError
if f.test(3.0) != 1003:
    raise RuntimeError

File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/overload_extend_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'overload_extend'

f = Overload_extend::Foo.new

raise RuntimeError if f.test(3) != 1
raise RuntimeError if f.test("hello") != 2
raise RuntimeError if f.test(3.5,2.5) != 6

